sbin/build.sh

```diff
@@ -837,6 +843,9 @@ getFirstTagFromOpenJDKGitRepo() {
   else
     echo "$firstMatchingNameFromRepo"
   fi
+
+  # Restore pwd
```

Comment:
Do you know which piece of code was in the wrong directory when the previous version went in? Would be useful to document it e.g. `# Without this, function blah is not in the right directory` especially if it's not possible for it to ensure it's in the right directory itself.

Reply:
i'll add a comment